notanorm/base.py

```diff
@@ -187,6 +191,19 @@ def __init__(self, *args, **kws):
         self._transaction = 0
         self._conn()
 
+    def __init_subclass__(cls: "DbBase", **kwargs):
+        if cls.uri_name:
+            cls.__known_drivers[cls.uri_name] = cls
+        cls.__known_drivers[cls.__name__] = cls
+
+    @classmethod
+    def get_driver_by_name(cls, name) -> Type["DbBase"]:
+        return cls.__known_drivers.get(name)
+
+    @classmethod
+    def uri_adjust(cls, args: List, kws: Dict):
```

Comment:
would be nice for this to have a docstring, mentioning the arg modification

Reply:
k done